### DISCUSSION TITLE
Expose types for the exposed components

### DISCUSSION THREAD

**Discussion by pikax** [histoire-dev/histoire#92] [Ideas]
`Story`, `Variant` and Controls components, should have their type exposed to be able to import them. 

Until https://github.com/vuejs/core/pull/3399 is not merged we could import a file that imports this automatically:

[Volar](https://github.com/johnsoncodehk/volar/tree/master/extensions/vscode-vue-language-features)
<img width="854" alt="image" src="https://user-images.githubusercontent.com/4620458/170001556-f1c68955-68e6-4f63-9965-619e40b4ede6.png">

**Comment by Akryum** [comment]
The controls component already have their types exposed from `@histoire/controls`

**Comment by Akryum** [comment]
@pikax Is it possible to type the slots?

**Reply by pikax** [reply]
Not yet waiting for https://github.com/vuejs/core/pull/2693 to be merged, after that we should be able to do it

**Comment by Akryum** [comment]
It's now implemented, see [setup](https://histoire.dev/guide/getting-started.html#typescript).
